### PR TITLE
LPS-161254 buildLang

### PR DESCRIPTION
**Pull request by p2kmgcl**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by p2kmgcl** [comment]
ci:test:sf

**Comment by p2kmgcl** [comment]
ci:test:relevant

**Comment by p2kmgcl** [comment]
ci:test:relevant

**Comment by p2kmgcl** [comment]
ci:test:sf

**Comment by p2kmgcl** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/Sidebar.js [hunk 1]
```diff
@@ -184,6 +200,75 @@ export default function Sidebar() {
...
+			document.body.addEventListener('mouseleave', stopResizing);
+			document.body.addEventListener('mouseup', stopResizing);
+		};
+
+		separatorElement.addEventListener('mousedown', handleMouseDown);
+
+		return () => {
+			stopResizing();
+			separatorElement.removeEventListener('mousedown', handleMouseDown);
+			document.body.removeEventListener('mousemove', handleMouseMove);
```
Comment (veroglez) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/Sidebar.js:268]: This is repeated inside the `stopResizing()`. Is this for some reason? I'm curious.

Comment (p2kmgcl) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/Sidebar.js:268]: That's a mistake, I am removing that :sweat_smile:

**Comment by veroglez** [comment]
😍 I love the resize! I just added a small comment.

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/Sidebar.js (hunk 1 above)
Comment (p2kmgcl) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/Sidebar.js]: Done :smile:

**Comment by p2kmgcl** [comment]
ci:forward pal Briasn

**Comment by p2kmgcl** [comment]
ci:forward

**Comment by ozysouza** [comment]
ci:stop

**Comment by ozysouza** [comment]
Please see https://github.com/brianchandotcom/liferay-portal/pull/122176

**Pull request closed without merging**